cmd/clusterctl/client/alpha/rollout_resumer.go

```diff
@@ -53,3 +67,11 @@ func resumeMachineDeployment(proxy cluster.Proxy, name, namespace string) error
 
 	return patchMachineDeployment(proxy, name, namespace, patch)
 }
+
+// resumeKubeadmControlPlane removes paused annotation.
+func resumeKubeadmControlPlane(proxy cluster.Proxy, name, namespace string) error {
```

Comment:
Should we also move this to `kubeadmcontrolplane.go`?

Reply:
as `resumeMachineDeployment` is also in the `rollout_resumer.go` I would rather keep it here or move both to their own files.

Reply:
I would propose to move both :-)